src/views/infra/zone/ZoneWizardZoneDetailsStep.vue

```diff
@@ -120,6 +120,43 @@
           }]"
         />
       </a-form-item>
+      <a-form-item
+        :label="$t('ip6cidr')"
+        v-bind="formItemLayout"
+        v-if="isAdvancedZone && securityGroupsEnabled"
+        has-feedback>
+        <a-input
+          v-decorator="['ipv6Cidr', {
+            rules: [
+              {
+                message: 'Please enter IpV6 CIDR',
+                initialValue: ip6cidr
+              }
+            ]
+          }]"
+        />
+      </a-form-item>
+            <a-form-item
+              :label="$t('ip6gateway')"
+              v-bind="formItemLayout"
+              v-if="isAdvancedZone && securityGroupsEnabled"
```

Comment:
Is this setting/field only taken into consideration when SG is enabled @GabrielBrascher ?

Reply:
IPv6 gateway and CIDR are handled on an advanced network with security group enabled. I don't see any harm in adding it to any type of zone network; however, that might lead users to believe that ACS will provide IPv6 support to the zone.